### PR TITLE
VACMS-7210 & VACMS-7217: Implement improved help text and fix help text duplication bug on service ief's

### LINKED ISSUE #7210: Implement improved help text for Vet Center service description, from #7189
## User Story or Problem Statement

See #7189

## Acceptance Criteria
- [ ] Help text from #7189 has been implemented to the Vet Center service form.

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [x] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

### LINKED ISSUE #7217: Vet Center service description help text is repeated
## Describe the defect
On the Edit Vet Center page, the help text under `general service description` appears twice. 

## To Reproduce

View example at: https://prod.cms.va.gov/node/3597/edit#edit-field-health-services

also in node view: https://staging.cms.va.gov/node/41246/edit

## Expected behavior
The help text appears once. 

## Screenshots 

Edit page: 
help text on Vet Center edit page

Node:
Vet Center help text duplication

 

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes #7210 
closes #7217

## Testing done
Visual

## Screenshots
image

## QA steps
 - [x] As an administrator, go to `node/3597/edit` and edit multiple existing services.
 - [x] Visually verify that each one has help text below `General service description` that reads `This standardized text will appear on your page automatically.`
 - [x] Visually verify help text below `Vet center service description` does not appear twice.

### CODE REVIEW

**Review by msbtterswrth — APPROVED** [review]

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
Just waiting on tests.  Please do not update branch.

File: docroot/modules/custom/va_gov_backend/js/displayServiceDescriptions.es6.js [hunk 1]
```diff
@@ -128,10 +128,18 @@
...
           const p = context.createElement("p");
+          const d = context.createElement("div");
           p.id = `${service.id}-services-general-description`;
           p.className = "services-general-description";
           p.textContent = "General service description";
+          d.className = "description";
+          d.id = `${service.id}-services-general-description-help`;
+          // Adding in help text for general description
+          d.textContent =
+            "This standardized text will appear on your page automatically.";
```
[the remark this answers is not in the document]
Comment (ethanteague) [reply docroot/modules/custom/va_gov_backend/js/displayServiceDescriptions.es6.js:139]: > I'm not a fan of text living here.
Yeah. I'm gonna add this to config. Good call, @swirtSJW

**Comment by ethanteague** [comment]
@swirtSJW closed out https://github.com/department-of-veterans-affairs/va.gov-cms/pull/7223 and handled it here. Also noting that theme suggestions aren't :100: yet with Entity Event Subscribers, so added the theme suggestion stuff via traditional hook system.

**Comment by suzanne-gray** [comment]
Hi Ethan! Tugboat isn't working for me right now, but based on the screen shot it looks like the help text for the Vet Center service description still needs to be updated. It should read:

Add another short paragraph and up to six bullet points detailing how your Vet Center provides this service.

Thanks!!

**Comment by ethanteague** [comment]
> Hi Ethan! Tugboat isn't working for me right now, but based on the screen shot it looks like the help text for the Vet Center service description still needs to be updated. It should read:
> 
> `Add another short paragraph and up to six bullet points detailing how your Vet Center provides this service.`
> 
> Thanks!!

Thanks for catching that, @suzanne-gray :+1: . Updated.

**Comment by suzanne-gray** [comment]
Looks perfect! Thanks and have a great weekend!

**Review by msbtterswrth — APPROVED** [review]
looks great!

**Comment by swirtSJW** [comment]
QA passes on this , Thanks for reworking the approaches Ethan this is much better.

@suzanne-gray   this help text strikes me as odd. 
image
"Add another short paragraph and up to six bullet points detailing how your Vet Center provides this service."

There is only one time they would be **adding**.   The other 95%+ of the time they are not adding.   The add "**another**",  makes the assumption they just added one, which most likely they did not.     Add another is language we use elsewhere for clicking a button to actually add another something.

**Comment by swirtSJW** [comment]
I moved the help text fix to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/7298 so that we can clear this for merge.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**